src/app/(components)/common/card.tsx

```diff
@@ -0,0 +1,18 @@
+import { HTMLAttributes } from "react";
+import { classNames } from "~/utils/classnames";
+
+export const Card: React.FC<
+  {
+    className?: string;
+  } & HTMLAttributes<HTMLDivElement>
+> = ({ className, ...rest }) => {
+  return (
+    <div
+      className={classNames(
+        "rounded border border-gray-750 bg-gray-850 shadow",
+        className
+      )}
+      {...rest}
```

Comment:
It took me a second to realize that this passes children. I would prefer for children to be an explicit target. At the very least I'd prefer this to self-close